### PR TITLE
enable `react/jsx-curly-brace-presence` rule and remove `react/jsx-no-literals` rule

### PR DESCRIPTION
**Pull request by B2o5T**
It's anti-practice in React as it's unnecessary JSX expressions and provokes confusing

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -34,10 +34,10 @@ module.exports = {
   },
 
   extends: [
-    'prettier',
     'plugin:import/typescript',
     'plugin:react/recommended',
     'plugin:react-hooks/recommended',
+    'prettier',
```
Comment (B2o5T) [inline .eslintrc.js:40]: ESLint applies configs cascading, and `eslint-config-prettier` **always** must be last!

**Comment by acao** [comment]
hmm, i think we should leave these for now to make i18n easier

**Comment by B2o5T** [comment]
@acao migration to i18n can be easy with [`@shopify/jsx-no-hardcoded-content`](https://github.com/Shopify/web-configs/blob/main/packages/eslint-plugin/docs/rules/jsx-no-hardcoded-content.md) eslint rule

**Review by thomasheyenbrock — APPROVED** [review]
I like this change! Imho it's the more "idiomatic" way of writing JSX. Regarding i18n, this isn't immediate priority right now so we can deal with this later. Also, the linter rule @B2o5T linked should help us with that 👍

**Pull request merged**